ietf/doc/views_conflict_review.py

```diff
@@ -392,10 +400,52 @@ def __init__(self, *args, **kwargs):
 
 @role_required("Secretariat","IRTF Chair","ISE")
 def start_review(request, name):
-    if has_role(request.user,"Secretariat"):
-        return start_review_as_secretariat(request,name)
+    '''Start the conflict review process.
+
+    Start the conflict review process, setting the initial 
+    shepherding AD, and possibly putting the review on a telechat.'''
+
+    def cleaned_notify(notify, doc):
+        if doc.stream.slug not in ['irtf', 'ise']:
+            return notify
+        stream_managers = [
+            re.sub('([^<]*<)?(.*)(>.*)', '\2', r)
+            for r in Recipient(slug='stream_managers').gather(streams=[doc.stream.slug])
```

Comment:
twitch: 
this really reads like it counts on side-effect. 
It might be better to create a new function in person/utils that returns the map of stream to manager addresses and have this code and the mailtrigger Recipient code use that instead. (It will make the explanation of the mailtrigger recipient more opaque, I'm really unsettled about Recipient being used this way.

Reply:
I'll pull that into its own function.

Jennifer and I had gone back and forth about whether the list of stream managers should be coded in Python, or whether it should be somehow retrieved from the database.  The best that we had come up with for retrieving it from the database was to find the group associated with the document stream, then find the chair for that group, and use that chair's email address.  Which approach would you like to see?